### ISSUE TITLE
Created lang on import breaks the BO

### ISSUE THREAD

**Issue by lovenunu** [thelia-modules/ImportT1#12]
If the import process has to create a lang, in ```ImportT1\Import\BaseImport line 194```, the BO crashes, trying to find the lang's flag asset image ```assets/img/.png``` as the lang has an empty code.

**Comment by roadster31** [comment]
In pre-1.5 versions of Thelia, the language code was missing, thus it is not possible to guess it during the import process.

You may raise this issue on T2, to protect the BO against such problems ?

**Comment by lovenunu** [comment]
Well, even if it isn't declared in the database's0constraints, having a lang without code nor locale is a consistency problem. Could we add a page to ask the user to link this unknown lang to an existing during the import process ?

**Comment by roadster31** [comment]
Why not ? But this will add a lot of complexity to te module.

**Comment by lovenunu** [comment]
Yes ... The temporary solution I used is to use the default lang instead of creating a new, but I know this doesn't cover all this module's needs.
